src/views/Settings.vue

```diff
@@ -286,7 +289,8 @@ export default defineComponent({
       orderLimitType: 'unlimited',
       fulfillmentOrderLimit: "" as number | string,
       facilityGroupDetails: {} as any,
-      isEComInvEnabled: false
+      isEComInvEnabled: false,
+      barcodeContentMessage: translate("Only allow received quantity to be incremented by scanning the barcode of products. If the identifier is not found, the scan will default to using the internal name.", { space: '<br /><br />' })
```

Comment:
"Only allow received quantity......"  should be changed to "Only allow shipped quantity"